### ISSUE TITLE
[BUG?] Translations are not shown on the admin area for a templateselect field

### ISSUE THREAD

**Issue by FerranMartin** [AnimalDesign/bolt-translate#110]
Hi!

First of all, if this is not a bug and it's my fault.. sorry :(

I just installed the extension and try to translate my pages contenttype. 
These pages have a `templateselect` field that is `is_translateable`. So when I go to the pages admin area I see a `template` tab where I can set the template variables, and I can set a different value for each locale. The translated value it's saved correctly, and if I go to the frontend page I can see the translation. 
But the problem comes when I reload the pages admin area, and I go again to the `template` tab. Then the translation value it's not loaded, instead I see the value of the default locale, so if I save again the page the translated value it's overrided with the default one.

Details
-------

 - Translate Version: 4.0.5
 - Install type: Extension Repository
 - Bolt Version: v3.2.9
 - PHP version: 7.1.3
 - Used webserver: Apache/2.4.25 (Ubuntu)

Reproduction
------------
Translate config file:
```
locales:
    ca_ES:
        label: Català
        slug: ca
    es_ES:
        label: Español
        slug: es
```

The Pages contenttype:
```
pages:
    name: Pages
    singular_name: Page
    fields:
        locale:
            type: locale
            group: content
        cadata:
            type: hidden
        caslug:
            type: locale_data
            index: true
        esdata:
            type: hidden
        esslug:
            type: locale_data
            index: true

        title:
            type: text
            class: large
            group: content
            is_translateable: true
        slug:
            type: slug
            uses: title
            is_translateable: true
        image:
            type: image
        body:
            type: html
            height: 300px
            is_translateable: true
        template:
            type: templateselect
            filter: 'page*.twig'
            is_translateable: true
#    taxonomy: [ groups ]
    recordsperpage: 100
```

I dunno if it's necessary, but here is the templatefields of the theme used in the pages. In concrete I'm trying the translation with a page that is using a 'page-manifest.twig' template.

```
templatefields:
    page-manifest.twig:
        title:
            type: text
            label: Título
        subtitle:
            type: text
            label: Subtítulo
        images:
            type: imagelist
            label: Imágenes
    page-resources.twig:
        resources:
            type: repeater
            label: Recursos
            fields:
                title:
                    type: text
                    label: Título
                url:
                    type: text
                    label: Enlace
    page-news.twig:
        news:
            type: repeater
            label: Noticias
            fields:
                title:
                    type: text
                    label: Título
                date:
                    type: date
                    label: Fecha
                url:
                    type: text
                    label: Enlace
                origin:
                    type: text
                    label: Medio de comunicación
```

## Here some images to illustrate the problem:

 - First time that I set the translation:

<img width="716" alt="captura de pantalla 2017-03-21 a las 10 52 21" src="https://cloud.githubusercontent.com/assets/1987244/24141862/121a43a6-0e25-11e7-8072-03e105c1ec5d.png">

 - The translation it's shown in the frontend:

<img width="784" alt="captura de pantalla 2017-03-21 a las 10 52 32" src="https://cloud.githubusercontent.com/assets/1987244/24141897/363ded78-0e25-11e7-9d56-22e2cc9a60d7.png">

 - When I reload the admin area the translation it's not showing:

<img width="715" alt="captura de pantalla 2017-03-21 a las 10 53 00" src="https://cloud.githubusercontent.com/assets/1987244/24141919/48fed1c0-0e25-11e7-9041-ff7c60f130e5.png">

 - But in the database the data it's correct:

<img width="940" alt="captura de pantalla 2017-03-21 a las 10 53 34" src="https://cloud.githubusercontent.com/assets/1987244/24141935/54f98de4-0e25-11e7-8155-86635c0644fd.png">

**Comment by jackiboy** [comment]
Hi! I am having the same issue as above. I have set the template select field to `is_translateable: true` and the saved changes appear on the frontend but reloading the edit page the changes do not appear to be saved.
------------
Bolt Version:  Bolt 3.2.11
Translate Version: 4.0.6

**Comment by benwallis** [comment]
Same issue here. Admin does not show different data for each locale - but it shows correct on front end (as long as not saved over accidentally).

Bolt version: 3.2.7
Translate version: 4.0.6
Labels version: 3.0.7

It's a simple one page website. Here is my content type definition:
```
sections:
    name: Sections
    singular_name: Section
    fields:
        locale:
            type: locale
            group: meta
        name:
            label: 'Section Name'
            type: text
            class: large
            readonly: true
        slug:
            type: slug
            uses: name
        template:
            type: templateselect
            is_translateable: true
            filter: '*_section.twig'
        frdata:
            type: hidden
        frslug:
            type: locale_data
            index: true
        endata:
            type: hidden
        enslug:
            type: locale_data
            index: true
        rudata:
            type: hidden
        ruslug:
            type: locale_data
            index: true
    sort: datecreated
    default_status: published
    viewless: true
    searchable: false
    icon_many: "fa:newspaper-o"
    icon_one: "fa:newspaper-o"
```
```
And my translate.animal.yml:
locales:
    en_GB:
        label: English
        slug: en
    fr_FR:
        label: Francais
        slug: fr
    ru_RU:
        label: русский
        slug: ru

routing_override: false
menu_override: false
url_generator_override: false
translate_slugs: false
use_accept_language_header: false
```

**Comment by SahAssar** [comment]
This is fixed in the latest release via #127 / #132 

Please reopen if I'm wrong :)
